examples/simple-non-wasm/programs/counter-rs/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "counter-rs"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4"
+bytes = "0.4"
+dotenv = "0.14"
+env_logger = "0.9"
+fuel-core = "0.6"
+fuel-gql-client = "0.6"
+fuel-tx = "0.9"
+fuels = "0.15"
+fuels-abigen-macro = "0.15"
+json = "0.12"
+log = "0.4"
+pretty_env_logger = "0.4"
+rand = "0.8"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tokio = { version = "1", features = ["full"] }
+
+[workspace]
```

Comment:
This lockfile is not included in the repo. This project needs not be included in the workspace (it's a separate little app only used for this specific example)

Reply:
Is it possible to include this within the workspace regardless?

If we are to not include this under the same workspace, we'll need to build it from scratch (including all it's dependencies) when testing locally or during CI.

Having it under the same workspace means they can share build artifacts which can speed up builds a lot! It also ensures that this example will get tested when running `cargo test` etc from the root of the repo.